airflow/www/views.py

```diff
@@ -307,7 +307,7 @@ def dag_stats(self, session=None):
 
         # Filter by get parameters
         selected_dag_ids = {
-            unquote(dag_id) for dag_id in request.args.get('dag_ids', '').split(',') if dag_id
+            unquote(dag_id) for dag_id in request.args.getlist('dag_ids[]') if dag_id
```

Comment:
```suggestion
            unquote(dag_id) for dag_id in request.args.getlist('dag_ids') if dag_id
```

There's no requirement on `[]` got `getlist`, and both of those characters get url-escaped which just makes the URL a bit ugly is all (and longer) :)

Reply:
+1 -> [] is just a strange convention and not a widely used one. I tried to find anyone using it and could not find it..

Reply:
PHP uses this syntax, but it's not very popular in the Python world. I will delete it.
https://stackoverflow.com/a/9547490/1849502

Reply:
I found the real specifications that describe this format.
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#style-examples last row.
![Screenshot 2020-02-03 at 21 39 32](https://user-images.githubusercontent.com/12058428/73689125-ceccf500-46cd-11ea-988a-fc72bad9a2ef.png)